### ISSUE TITLE
Listening history not updated on Spotify account

### ISSUE THREAD

**Issue by buongiorgio** [KRTirtho/spotube#940]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When I listen to music in Spotube, the listening history is not updated on my account (premium).

### Expected Behavior

History is updated.

### Steps to reproduce

1. Listen to music on Spotube.
2. Open the official app.
3. Go to account > Listening history
4. The tracks I just listened to are not in my history.

### Operating System

Android 14 (CalyxOS)

### Spotube version

v3.3.0 build number 26

### Installation source

F-Droid (Android)

**Comment by KRTirtho** [comment]
Read the readme first?

- **🔉 Playback control is done locally, not on the server**

**Comment by buongiorgio** [comment]
Read a dictionary first?

**Comment by KRTirtho** [comment]
Excuse me, what?

**Comment by buongiorgio** [comment]
You clearly don't have a good command of the English language, as you tell people to read a document that doesn't address the issue, so you should read a dictionary first.

**Comment by KRTirtho** [comment]
Oh, don't worry about that. I've not only read the dictionary but also annotated it with cross-references and footnotes. My lexicon is so labyrinthine that even lexicographers would find themselves lost in the convoluted intricacies of my linguistic prowess. Care for a syntactic labyrinth challenge, or should I just elucidate the ineffable in plain English?
Ever wondered why programming languages fascinate me? It's simple, an insatiable curiosity to explore diverse spoken languages fuels my love for the coding realm. Thus, don't doubt my dictionary knowledge.

Btw, I didn't write the readme 😅

**Comment by buongiorgio** [comment]
I know a lot of bangla, they usually attend cattle. They are good with that. Very weird finding out one who is into programming. So you pretend to know so many words in English. Well, if that is the case, then probably is challenging for you to use logic.

**Comment by KRTirtho** [comment]
Ok. That's just racism. Till now, I was kidding but this is rude and unethical.

You're the one who doesn't understand simple English and then questioning my English. You accused me of not answering and instead I redirect everyone to readme. Although I clearly explained why Spotube doesn't support Spotify listening history.

Racism is sick. And it's pathetic when someone is so clueless and incapable that their final resort to regain their respect is to culturally abuse another person and point up their delusional stereotypes about it.

**Comment by buongiorgio** [comment]
I see, now you're playing the race card. Poor snowflake. Again, you are weak in logic, you rudely tell the people to RTFM, but in TFM the issue is not addressed. So next time be polite, double check what you suggest to read and GFY.

**Comment by KRTirtho** [comment]
Nobody ever plays the "race card" there's only racist like you and people who aren't racist. You directly insulted my culture and my people. I guess your race will be really proud of that and you just let me judge and guess in what kind of slum you are from. No no, even people in slum are better than you. People like you deserve much worse than that. Since you're so proud of your race, tell us all about that.

Also, addressing the weakness in logic I can clearly see who's the weak and ignorant one. And telling people to read the README is not a rude behavior and also your issue was addressed in the RAEADME but in different words and normal people understands this all the time.

I deliberately work on this project all the time, completing bugs and useful feature requests, lazy-ass people like you can't even read the freaking README first and creates useless issues like this. And to save time when I brought up their mistake, I'm rude.

> So next time be polite, double check what you suggest to read and GFY.

Please look up the dictionary and learn the meaning of politness. Because in your whatever race, people aren't polite judging from your poor behavior.

**Comment by buongiorgio** [comment]
Uh, the boy is really mad at me! I just complimented your people for being so good in the art of attending cattle: there's something wrong with that job? And you just said "people in the slum are better than you". Living in a slum is not a shame, unless you think people who's got a little money is better. You don't respect people who does manual jobs and is not rich, so, you're the racist. And tell people to RTFM is rude, if you are not aware of that you should educate yourself on IT culture.

**Comment by KRTirtho** [comment]
> I know a lot of bangla, they usually attend cattle. They are good with that. Very weird finding out one who is into programming. So you pretend to know so many words in English. Well, if that is the case, then probably is challenging for you to use logic.

This tone doesn't indicate compliment and I might not be the brightest, but I can differentiate between compliment and disrespect. 

Also, don't try to turn the tables, making me into thinking I disrespect work that involve manual labour. I thought you were just a low life racist, you're even worse. A manipulator. Please seek professional help.

> And tell people to RTFM is rude, if you are not aware of that you should educate yourself on IT culture.

I've not opened up an overseas tech support here. And reading the README first is a basic requirement before opening an issue. It saves time for both parties. It's a common sense in the Open Source community. I even took time to point out the part that addressed your issue while also explaining it. And you call that rude? I don't know where you're from, but you've never experienced actual rudeness.

> Uh, the boy is really mad at me!

I'm not mad at you, I pity you. And am slightly disappointed.

**Comment by buongiorgio** [comment]
You're so funny! In your words I'm a "manipulator" and i should "seek professional help". You just showed your true self: for you saying "people in the slum are better than you" is some kind of insult. And I manipulated you through a couple of messages in an internet forum? If that's the case, you should seek professional. BTW, is this (cr)app respecting Spotify ToS? IMO no...
